modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp

```diff
@@ -93,8 +93,8 @@ CommerceAddress shippingAddress = commerceShipmentDisplayContext.getShippingAddr
 					'/commerce.commercecountry/get-shipping-commerce-countries',
 					{
 						companyId: <%= company.getCompanyId() %>,
-						end: -1,
-						start: -1,
+						shippingAllowed: true,
+						active: true,
```

Comment:
@riccardo-alberti I think we should sort these.

Reply:
hey @dsitu the service method is 
public List<CommerceCountry> getShippingCommerceCountries(
		long companyId, boolean shippingAllowed, boolean active) {

Reply:
@riccardo-alberti ohhh ic. Yeah we should follow the original method order then. Thanks for clarifying!